Update Manual File Locking (#4509)
owncloud/docs: modules/user_manual/pages/files/manual_file_locking.adoc

= Manual File Locking
:toc: right

== Introduction

Manual file locking allows users, if enabled by the ownCloud administrator, to lock files in shared areas while working on them in order to prevent concurrent changes from other users (check-in/check-out). 

> All storages are supported as locking takes place on the WebDAV level. The locks will only be available via ownCloud, not when a user works directly on the storage. Using the context menu of files, every user who has access can lock them. Users can recognize locked files by the means of a new lock indicator. While a file is locked, other users can still access it but they can not make any changes. Locked files can manually be unlocked by the lock owner (the user who locked the file; exclusive locking) using the "Locks" tab in the file details view (right sidebar).

Reviewer: ```suggestion
All storages are supported as locking takes place on the WebDAV level. The locks will only be available via ownCloud, not when a user works directly on the storage. Using the context menu of files, every user who has access can lock them. Users can recognize locked files by the means of a new lock indicator. While a file is locked, other users can still access it but they cannot make any changes. Locked files can manually be unlocked by the lock owner (the user who locked the file; exclusive locking) using the "Locks" tab in the file details view (right sidebar).
```

== Using Manual File Locking

> NOTE: The _user-facing_ components in the web interface are disabled by default and must be enabled by the ownCloud admin, because this feature allows users to lock other user's files *exclusively*. Even the owner of the file can't unlock them, only the locking user can unlock or if the lock expires.

Reviewer: ```suggestion
NOTE: The _user-facing_ components in the web interface are disabled by default and must be enabled by the ownCloud admin, because this feature allows users to lock other users files *exclusively*. Even the owner of the file can't unlock them, only the locking user can unlock until the lock expires.
```
